geode-junit/src/main/java/org/apache/geode/internal/statistics/StatUtils.java

```diff
@@ -42,6 +42,8 @@
    */
   public static void compareStatArchiveFiles(final File expectedStatArchiveFile,
       final File actualStatArchiveFile) throws IOException {
+    System.out.println(actualStatArchiveFile);
+    System.out.println(expectedStatArchiveFile);
```

Comment:
Are these new System.out.printlns intentional?

Reply:
Whoops! Didn’t mean to check that in. Thanks!